README.md

```diff
@@ -47,7 +47,7 @@ impl HelloService for HelloServer {
 
 fn main() {
     let server_handle = HelloServer.spawn("0.0.0.0:0").unwrap();
-    let client = hello_service::Client::new(server_handle.local_addr()).unwrap();
+    let client = hello_service::Client::new(server_handle.dialer()).unwrap();
```

Comment:
Maybe we should have the example in the readme just use a specific IP address? Like localhost:10000 or something? Then people don't have to learn about dialers if they don't care about them.